### PR TITLE
RxBumper Subpod!

### PR DESCRIPTION
**Pull request by elikohen**
Ueah!! I'm back! 😂 

I've just added a subpod called RxBumper on our fork to add rx observing functionalities to bumper. Also the example is updated with the new pod and some examples on how to use it.

Note: This code parts from code in ABIOS-3097-bumper-swift4. So i encourage that you first merge https://github.com/letgoapp/bumper/pull/9 pr  into your code and then check this one ;)

Note2: As bumper is downcase all code is on bottom after the pods :D

### CODE REVIEW

File: Example/bumper/BumperFeatures.swift [hunk 1]
```diff
@@ -9,6 +9,9 @@
 
 import Foundation
 import bumper
+#if (RX_BUMPER)
```
Comment (ahernandezlopez) [inline Example/bumper/BumperFeatures.swift]: nice that is optional :)

File: Example/Pods/Local Podspecs/bumper.podspec.json [hunk 2]
```diff
@@ -21,9 +21,32 @@
...
+        ],
+        "RxCocoa": [
+          "~> 4.0.0"
+        ]
+      },
+      "xcconfig": {
+        "OTHER_SWIFT_FLAGS": "$(inherited) -D RX_BUMPER"
+      }
+    }
+  ]
```
Comment (ahernandezlopez) [inline Example/Pods/Local Podspecs/bumper.podspec.json]: love the subpod setup

**Comment by ahernandezlopez** [comment]
https://github.com/letgoapp/bumper/pull/9 was merged yday :)

File: Example/bumper/BumperFeatures.swift (hunk 1 above)
Comment (elikohen) [reply Example/bumper/BumperFeatures.swift]: Yep, if you just use the bumper pod, no rx is imported nor used or exposed

**Pull request merged**